### PR TITLE
[BUG]add len(items) > 0 for getLatestSecond function

### PR DESCRIPTION
**Pull request by zsy619**
### Does this pull request fix one issue?
#108 
### Special notes for reviews

### CODE REVIEW

File: core/log/metric/reader.go [hunk 1]
```diff
@@ -174,7 +175,7 @@ func readLine(bufReader *bufio.Reader) (string, error) {
 }
 
 func getLatestSecond(items []*base.MetricItem) uint64 {
-	if items == nil {
+	if items == nil || len(items) == 0 {
```
Comment (louyuting) [inline core/log/metric/reader.go:178]: len(items) == 0  is OK.
The length of nil slice is 0.

Comment (hellosmallstone) [reply core/log/metric/reader.go:178]: Yes, the length of nil slice is 0. 
However, the zero value for slice is nil. It seems necessary to standardize the way to return an empty slice in the project. Even in the same function, there are different ways to return empty slice, such as function ReadMetricsByEndTime in this file.

Comment (louyuting) [reply core/log/metric/reader.go:178]: Yes, your idea is great.
Could you please d push a PR to enhance.

**Comment by louyuting** [comment]
@zsy619  Any update?

**Pull request merged**